.github/ISSUE_TEMPLATE/feature_request_beta.yaml

```diff
@@ -0,0 +1,105 @@
+---
+name: ✨ Feature Request Beta
+description: Propose a new Nautobot feature or enhancement
+labels: ["type: feature"]
+body:
+  - type: markdown
+    attributes:
+      value: >
+        **NOTE:** IF YOUR ISSUE DOES NOT FOLLOW THIS TEMPLATE, YOU WILL BE REQUIRED TO
+        UPDATE THE ISSUE IN ORDER TO HAVE IT REVIEWED. This form is only for proposing
+        specific new features or enhancements. If you have a general idea or question,
+        please start a [discussion](https://github.com/nautobot/nautobot/discussions) instead.
+  - type: dropdown
+    id: persona
+    attributes:
+      label: As a WHO, the user persona___,
+      description: Please choose the appropriate [persona](https://github.com/nautobot/nautobot/wiki/Personas)
+      multiple: false
+      options:
+        - "Ozzie - Operator"
+        - "Nelly - Network Engineer"
+        - "Austin - Network Automation Engineer"
+        - "Tammy - Tools Integrator"
+        - "Mary - Manager"
+        - "Patti - Platform Admin"
+        - "Isabella - Infrastructure Owner"
+        - "Dan - Data Owner"
+        - "P.D. - Plugin Developer"
+        - "Cora - Coder"
+    validations:
+      required: true
+  - type: textarea
+    id: what
+    attributes:
+      label: I want WHAT
+      description: Please enter the desired functionality to be implemented.
+      placeholder: To develop Nautobot plugins that can be inserted as a top-level item in the navigation bar
+    validations:
+      required: true
+  - type: textarea
+    id: why
+    attributes:
+      label: So that WHY
+      description: Please enter the reason or value for the desired functionality.
+      placeholder: >
+        My plugin is seen as first-class citizen by my users and so that the
+        plugins drop-down menu isn't overloaded. When I use Nautobot as an App Platform,
+        it'll also provide more focus on the Plugins/Apps than core SoT drop-downs (for
+        users that aren't permitted to view those items)
+    validations:
+      required: true
+  - type: textarea
+    id: acceptance_criteria
+    attributes:
+      label: I know this is done when... HOW
+      description: The requirements/acceptance criteria (a list) as to how this functionality should be implemented.
+      placeholder: |
+        - I create a Nautobot plugin and have the option to inject it in the Plugins
+          dropdown menu or create a new dedicated dropdown in the navigation bar.
+        - When multiple plugins provide dedicated dropdowns, I can re-order the plugin
+          dropdowns in the navbar.
```

Comment:
```suggestion
        - I create a Nautobot plugin and have the option to inject it in the Plugins
          dropdown menu or create a new dedicated dropdown in the navigation bar.
          
        - When multiple plugins provide dedicated dropdowns, I can re-order the plugin
          dropdowns in the navbar.
```

Still appears to be rendering run together. Does separating this with another newline make them appear as a list in the text area?

<img width="1146" alt="image" src="https://user-images.githubusercontent.com/31187/155432172-b07330ae-9e61-4845-8c40-ddb0ad3ce680.png">

Reply:
This is an issue with Safari and Placeholders. https://github.com/github/feedback/discussions/4290

Reply:
More information about the issue can be found https://github.com/tests-always-included/angular-placeholder/issues/14#issuecomment-202400788